units/6_unit/05b_lesson/starter_code/inventory.txt

```diff
@@ -0,0 +1,163 @@
+brand,size,material,auto,color,stars,price
```

Comment:
Since `project.md` calls this out as a “comma-separated value” file, please change the extension to `.csv`.